javascript-conditionals/main.js

```diff
@@ -36,47 +36,51 @@ function startsWithJ(string) {
 console.log('Starts with J ', startsWithJ('Cat'));
 
 function isOldEnoughToDrink(person) {
-  const Brad = {
+  const brad = {
     age: 25,
     name: 'Brad Long'
   };
-  const over = Brad.age;
-  if (over > 21) {
+  const over = brad.age;
+  if (over >= 21) {
     return true;
-  } else { return false; }
+  } else {
+    return false;
+  }
 }
 console.log('Is person old enough to drink? ', isOldEnoughToDrink());
 
 function isOldEnoughToDrive(person) {
-  const Homer = {
+  const homer = {
     name: 'Homer Simpson',
     age: 30
   };
-  const old = Homer.age;
-  if (old > 16) {
+  const old = homer.age;
+  if (old >= 16) {
     return true;
   } else { return false; }
 }
 console.log('Is old enough to drive ', isOldEnoughToDrive());
 
 function isOldEnoughToDrinkAndDrive(person) {
-  const Homer = {
-    name: 'Homer Simpson',
+  const homer = {
+    name: 'homer Simpson',
     age: 1
   };
-  const nogo = Homer.age;
+  const nogo = homer.age;
   if (nogo < 10) {
```

Comment:
If you're 10, you wouldn't be able to either drink or drive. You should be checking for 2 separate numbers here.